invenio_cli/commands/requirements.py

```diff
@@ -123,9 +125,10 @@ def check_docker_version(cls, major, minor=-1, patch=-1, exact=False):
     def check_docker_compose_version(cls, major, minor=-1, patch=-1, exact=False):
         """Check the docker compose version."""
         # Output comes in the form of
-        # 'docker-compose version 1.27.4, build 4484c46d9d\n'
+        # 'Docker Compose version v2.17.3\n'
+        docker_helper = DockerHelper("", local=False)
         try:
-            result = run_cmd(["docker-compose", "--version"])
+            result = run_cmd(docker_helper.docker_compose + ["version"])
```

Comment:
```suggestion
            result = run_cmd("docker", "compose", "version"])
```
wouldn't that work?

Reply:
Yes, but as discussed IRL, let's shelve for now and merge since it might be a bigger task to remove the helper usage across the module.